models/extractor_reports/extractor_reports.yml

```diff
@@ -227,13 +227,13 @@ models:
       - name: fiscal_period
         description: Fiscal year/period
       - name: debit_amount
-        description: Total debit postings
+        description: Total debit postings, derived from a balance carried forward field, will match accumulated balance and turnover amounts if the carryover was designated a debit
```

Comment:
I see how the intentional duplicated columns are shown [here](https://github.com/fivetran/dbt_sap/blob/a21727760967de9aaa09e5b2991223c09c99491b/models/extractor_reports/intermediate/int_sap__0fi_gl_10_sums.sql#L60-L61) in the intermediate model. However, can you share how the fields from this intermediate model end up impacting the results of these columns in the end model.

Reply:
The description is helpful, but we should also include the original names as that may be more insightful to SAP users.